test/push_images.sh

```diff
@@ -1,10 +1,6 @@
 #!/bin/bash
 
-for i in {1..10}; do
-	docker login -u username -p password localhost
-	if [ $? -eq 0 ]; then
-		break
-	fi
+while ! docker login -u username -p password localhost; do
```

Comment:
Does this really belong in this PR? I'm fine if you want to earmark it in, just wand to double check.

Reply:
Maybe not, it just seemed like the easiest fix for `$i` being unused. I could swap it out to just echo the iteration. What do you think?